src/main/java/org/mule/extension/http/internal/listener/profiling/tracing/HttpListenerCurrentSpanCustomizer.java

```diff
@@ -41,16 +46,24 @@ public class HttpListenerCurrentSpanCustomizer extends HttpCurrentSpanCustomizer
   private final String host;
   private final int port;
 
-  private HttpListenerCurrentSpanCustomizer(HttpRequestAttributes attributes, String host, int port) {
+  //W-12558102
```

Comment:
I would remove the comment for the feature.

Reply:
Done. Should I remove all the other similar comments?